Formula/libvmaf.rb

```diff
@@ -12,18 +12,31 @@ class Libvmaf < Formula
     sha256 "c5b2cbf13a844a4591e2f1dbf7d20266715130802ba3b030f45e9471da994e86" => :sierra
   end
 
+  depends_on "meson" => [:build, :test]
+  depends_on "ninja" => [:build, :test]
```

Comment:
Are these needed for the `:test` stage? Looks like in the test you just make use of the C compiler and not meson and ninja.

Reply:
Good point. There is a Ninja test command in libvmaf but it requires the source files to be present. I have not used tests in Homebrew before, so I don't know how to access these — should I copy them over during installation?

Reply:
I think what you have is fine. Tests in Homebrew don't need to have particularly high coverage - in the end what we're testing is whether we have compiled it correctly with the needed runtime dependencies (where applicable). We don't really have the resources to comprehensively test whether every feature of every software in Homebrew works.

I'd say make like this: 
```suggestion
  depends_on "ninja" => :build
```
and the same with `meson` and that'll be enough.

Reply:
Changed as suggested, thanks for the feedback.